hyrax/app/inputs/nested_affiliation_input.rb

```diff
@@ -24,14 +24,15 @@ def build_components(attribute_name, value, index, options, parent=@builder.obje
     field_id = id_for(attribute_name, index, field, parent)
     field_value = value.send(field).first
     field_class = class_for(attribute_name, field)
+    field_requirements = requirements_for(attribute_name, field)
 
     out << "  <div class='col-md-3'>"
-    out << template.label_tag(field_name, field.to_s.humanize, required: required)
+    out << template.label_tag(field_id, field.to_s.humanize, required: required)
```

Comment:
yes, I know the rails docs say "name" in the label_tag arguments.  They don't mean "the html element called name" they mean "what you want the name of your for field to be" and yes, I agree, that's a pretty poor excuse